typescript/src/schema/api.ts

```diff
@@ -193,6 +195,17 @@ export class API {
       '  '
     );
   }
+
+  get dump() {
```

Comment:
This is a new getter to produce an API dump.

Reply:
Just to clarify, it will be only used in tests or during debugging. We won't dump this thing for regular libraries.